nikola/__init__.py

```diff
@@ -29,8 +29,15 @@
 import os
 import sys
 
+# The current Nikola version:
 __version__ = '8.3.0'
+# A flag whether logging should emmit debug information:
 DEBUG = bool(os.getenv('NIKOLA_DEBUG'))
+# A flag whether special templates trace logging should be generated:
+TEMPLATES_TRACE = bool(os.getenv('NIKOLA_TEMPLATES_TRACE'))
+# When this flag is set, fewer exceptions are handled internally;
+# instead they are left unhandled for the run time system to deal with them,
+# which typically leads to the stack traces being exposed.
```

Comment:
```suggestion

# A flag to show tracebacks of unhandled exceptions.
# An alternative to the DEBUG flag with less noise.
```

(`NIKOLA_DEBUG` used to be very spammy with yapsy, it’s still not great to have it always on; `export NIKOLA_SHOW_TRACEBACKS=1` is in my `.zshrc` though.)

Reply:
The wording "of unhandled exceptions" suggests whether the exceptions is handled or not is independent of this setting. As far as I remember, this is not the case.

Reply:
Technically, all exceptions are handled. This setting affects exceptions that bubble up to the global handler and therefore crash the process.